test_suite/generic/test_generic.py

```diff
@@ -14,6 +14,14 @@ def test_bash_history_is_empty(self, host):
                 file_content_length = len(bash_history_file.content_string)
                 assert file_content_length == 0, f'{file_path} must be empty or nonexistent'
 
+    @pytest.mark.run_on(['rhel'])
```

Comment:
I would add fedora to the test. And also investigate centos

Reply:
I added a TODO for the future, since it's not in the priorities now.